integration-tests/bats/dump.bats

```diff
@@ -177,7 +201,7 @@ teardown() {
     dolt add .
     dolt commit -m "create tables"
 
-    run dolt dump
```

Comment:
Any reason to use --no-batch here?

Reply:
With batching we get a `CREATE TABLE 'dolt_procedures'` line that will error out upon import.